src/main/scala/org/dbpedia/spotlight/wikistats/ComputeStats.scala

```diff
@@ -45,14 +44,14 @@ class ComputeStats(lang: String) (implicit val sc: SparkContext,implicit val sql
 
   def buildCounts(wikipediaParser: JsonPediaParser,stopWordLoc: String): (DataFrame, DataFrame)={
 
-
     val allSfs = wikipediaParser.getSfs().collect().toList
-
+    println ("All sfs count")
```

Comment:
Maybe it would be worth to change the prints to log statements, it gives a better control for debuging and what not